TheaHansen/FE-Hovedopgave-2024-2025
# Merging h2-52 into vite-dev

Hey Frederikke! 

Test to get into the page:
1. Add items to basket
2. Click on basket icon
3. Click on button in dropdown
4. You'll get into the checkout page

Steps to check:

- Must have a form for customer information
- Must show overview of products added to the basket
- Must have shipment options
- Must have payment options
- Must have checkbox for accepting terms and conditions
- If added more than 6 different products, the overview of products is getting a scrollbar
- You can only choose either PostNord or GLS. The same with payment method.
